astroquery/gemini/core.py

```diff
@@ -462,6 +730,76 @@ def _gemini_json_to_table(json):
     return data_table
 
 
+def _gemini_cal_html_to_table(html):
+    """
+    takes a HTML table list of calibrations as returned from the Gemini archive webserver and turns it into an
+    `~astropy.table.Table`
+
+    Parameters
+    ----------
+    html : str
+        A HTML table with the calibration records
+
+    Returns
+    -------
+    response : `~astropy.table.Table`
+    """
+
+    data_table = Table(masked=True)
+
+    table = BeautifulSoup(html, features="html.parser").find("table")
+    # This is future-proofing, the existing page has a bug putting the header inside the body
+    # but if that gets fixed, we'll want this
+    cols = None
+    table_header = table.find("thead")
```

Comment:
This has to handle `NoneType` as an option for `table`, at least the test failure seems legit to me.